### ISSUE TITLE
More detailed accessibility assessment

### ISSUE THREAD

**Issue by cacology** [scholarslab/clockwork#74]
Issue #58 did a simple assessment, but one in more detail could be helpful. See that issue for some suggested tools.

**Comment by cacology** [comment]
FireEyes suggests going through the site with a screen reader: "Congratulations! No accessibility violations found. Now you should perform manual testing using assistive technologies like NVDA, VoiceOver and JAWS."
